p5/sketch/userspace.py

```diff
@@ -283,19 +283,13 @@ def exit(*args, **kwargs):
     that necessary cleanup steps are performed before exiting the
     sketch.
 
-    :param args: positional argumets to pass to Python's builtin
-        `exit()` function.
+    :param args: positional arguments are ignored.
```

Comment:
We can probably remove both `*args` and `**kwargs`, since we are not calling the python native `exit` anyway, and Processing's Java API doesn't take any arguments for the `exit` function either: https://processing.org/reference/exit_.html